src/ceph-volume/ceph_volume/devices/simple/activate.py

```diff
@@ -135,6 +136,13 @@ def activate(self, args):
         with open(args.json_config, 'r') as fp:
             osd_metadata = json.load(fp)
 
+        self.is_encrypted = osd_metadata.get('encrypted', False)
```

Comment:
While trying to reproduce, both `journal` and `journal_dmcrypt` are present in a dmcrypt OSD deployed with ceph-disk:

```
# cat /etc/ceph/osd/1-71e57981-4a96-47ad-8020-27de98bb0b7d.json 
{
    "active": "ok", 
    "ceph_fsid": "c90755e4-9a50-4a2f-909e-f633ebb0c8e0", 
    "cluster_name": "ceph", 
    "data": {
        "path": "/dev/sdac1", 
        "uuid": "71e57981-4a96-47ad-8020-27de98bb0b7d"
    }, 
    "encrypted": true, 
    "encryption_type": "luks", 
    "fsid": "71e57981-4a96-47ad-8020-27de98bb0b7d", 
    "journal": {
        "path": "/dev/sdac2", 
        "uuid": "537d18af-2c5b-4cbb-bf37-8e0c15da2e59"
    }, 
    "journal_dmcrypt": {
        "path": "/var/lib/ceph/osd/ceph-1/journal_dmcrypt", 
        "uuid": "537d18af-2c5b-4cbb-bf37-8e0c15da2e59"
    }, 
    "journal_uuid": "537d18af-2c5b-4cbb-bf37-8e0c15da2e59", 
    "keyring": "AQDEacldD50TFBAAzLOxIMQuVw0P0/UMACncLQ==", 
    "lockbox.keyring": "AQDJacldbU4wGhAAWqJRXFNLUIvn4GmusrutuA==", 
    "magic": "ceph osd volume v026", 
    "ready": "ready", 
    "systemd": "", 
    "type": "filestore", 
    "whoami": 1
}
```